solvation_analysis/analysis_library.py

```diff
@@ -140,11 +140,15 @@ def find_shells(self, shell_dict):
 
 class Coordination:
     """
-    Calculate the coordination number of each solvent.
+    Calculate the coordination number for each solvent.
 
     Coordination calculates the coordination number by averaging the number of
     coordinated solvents in all of the solvation shells. This is equivalent to
-    the typical method of integrating the RDF up to the solvation radius cutoff.
+    the typical method of integrating the solute-solvent RDF up to the solvation
+    radius cutoff. As a result, Coordination calculates species-species coordination
+    numbers, not the total coordination number of the solute. So if the coordination
+    number of mol1 is 3.2, there are on average 3.2 mol1 residues within the solvation
+    distance of each solute.
```

Comment:
You can now leverage your tutorial example as an easy inline example!

Reply:
Not sure how many lines is appropriate to set up the example, but showing the solution definition and run without explicitly defining the atom groups felt like a good compromise. I decided to demonstrate through a Solution object because that is how it is intended to be used. How does this look?

Reply:
Looks good to me. :)